### PR TITLE
[ASCII-1000] Fix nolint for ASC

### PR DESCRIPTION
**Pull request by pgimalac**
### What does this PR do?

Fix `//nolint //TODO(ASC)`, and a few other `//nolint` while I was at it.

### Motivation

Those were temporary comments added to re-enable `revive`, removing them is better.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: cmd/agent/subcommands/run/command.go [hunk 1]
```diff
@@ -20,6 +19,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/DataDog/datadog-agent/comp/core/workloadmeta/defaults"
```
Comment (dustmop) [inline cmd/agent/subcommands/run/command.go:22]: Could you move this line lower so it lives with the rest of the "github.com/DataDog" imports.

File: pkg/cli/subcommands/check/command.go [hunk 2]
```diff
@@ -12,14 +12,15 @@ import (
...
 	"fmt"
-	"github.com/DataDog/datadog-agent/comp/core/workloadmeta/defaults"
 	"os"
 	"path/filepath"
 	"runtime"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/DataDog/datadog-agent/comp/core/workloadmeta/defaults"
```
Comment (dustmop) [inline pkg/cli/subcommands/check/command.go:22]: Could you move this lower to live with the other "github.com/DataDog" imports?

**Review by dustmop — APPROVED** [review]
LGTM, some small nits

**Review by AliDatadog — APPROVED** [review]

**Review by juliendoutre — APPROVED** [review]
Looks good to me for @DataDog/software-integrity-and-trust

**Review by dianashevchenko — APPROVED** [review]

**Pull request merged**